README.md

```diff
@@ -1,14 +1,27 @@
-# Project
+# Python environment tools for Visual Studio Code
 
-> This repo has been populated by an initial template to help get you started. Please
-> make sure to update the content to build a great experience for community-building.
+Performant Python environment tooling and support, such as locating all global Python installs and virtual environments.
 
-As the maintainer of this project, please make a few updates:
+## Environment Types Supported 
 
-- Improving this README.MD file to provide a great experience
-- Updating SUPPORT.MD with content about this project's support experience
-- Understanding the security reporting process in SECURITY.MD
-- Remove this section from the README
+- Windows Registry
+- Windows Store
+- PyEnv
+- Conda
+- PipEnv
+- Homebrew
+- VirtualEnvWrapper
+- Venv
+- VirtualEnv
+
+## Features 
+
+- Discovery of all global Python installs
+- Discovery of all Python virtual environments
+
+## Key Methodology
+
+We leverage Rust's performance to quickly scan for Python environments available to the user. When we find them, we gather all relevant information immediately to avoid the need for repeated discovery.
```

Comment:
Would like to call out out methodology and "principles" used in this work. Will likely need some additional details to ensure this is comprehensive